src/Exception/EntityDuplicateException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Szemul\Database\Exception;
+
+use Exception;
+use RuntimeException;
+
+class EntityDuplicateException extends RuntimeException
```

Comment:
This should probably extend DatabaseException. Or we should define an interface for all db exceptions and implement that. But the point is a catch DatabaseException should catch all exceptions thrown by the db module